### PR TITLE
feat(context selector): new context selector

### PR DESCRIPTION
**Pull request by matthewcarleton**
## Description
This is newly implemented context selector. It differs from the original design but this direction was taken from @kybaker and @mcarrano. The designs will be updated to reflect these changes.

@jeff-phillips-18 Can you confirm that I've build the test page out properly? I chose to add conditional logic in the navbar-vertical.html file rather than create an entirely new navbar file just for context selector.

For the context selector I've added a class of `navbar-pf-contextselector` to the nav-bar. I've also added `hidden-sm hidden-xs` to the `navbar-brand` rather than in the CSS to allow the dev to choose at which point to remove the logo for smaller screens. I've chosen these specifically because the Patternfly logo is so wide, but for smaller logos it could be removed at a smaller size. 

I've spoken to @mcarrano about adding instructions in the code tab on patternfly.org to properly guide the dev in implementing the context selector.

https://rawgit.com/matthewcarleton/patternfly/context-selector-dist/dist/tests/context-selector-vertical-nav.html

## PR checklist (if relevant)

- [ ] **Cross browser**: works in IE9
- [ ] **Cross browser**: works in IE10
- [x] **Cross browser**: works in IE11
- [x] **Cross browser**: works in Edge
- [x] **Cross browser**: works in Chrome
- [x] **Cross browser**: works in Firefox
- [x] **Cross browser**: works in Safari
- [x] **Cross browser**: works in Opera
- [x] **Responsive**: works in extra small, small, medium and large view ports.
- [x] **Preview the PR**: An image or a URL for designer to preview this PR is provided.

### CODE REVIEW

**Review by srambach — DISMISSED** [review]
Looks good to me. Documentation on the code tab would be great!

File: tests/pages/context-selector.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+categories: [Widgets]
+css-extra: false
+layout: page
+resource: true
+contextselector: true
+navindex: 1
+title: context selector
```
Comment (jeff-phillips-18) [inline tests/pages/context-selector.html]: Title case: `Context Selector`

**Review by jeff-phillips-18 — DISMISSED** [review]
Test pages structure look good! Just one small nit.

**Review by mcarrano — DISMISSED** [review]
Looks great. Thanks for all you work on this @matthewcarleton

**Review by jeff-phillips-18 — APPROVED** [review]

**Pull request merged**